client/trino-client/src/main/java/okhttp3/internal/tls/LegacyHostnameVerifier.java

```diff
@@ -13,20 +13,29 @@
  */
```

Comment:
> Remove OkHostnameVerifier usage

Did you mean inline `OkHostnameVerifier`? If so then please update the commit message. Also please mention `Util.verifyAsIpAddress`